.github/workflows/homebrew-formula.yml

```
@@ -0,0 +1,32 @@
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22.21.1"
+
+      - name: Sync formula from npm tarball
+        run: bash scripts/sync-homebrew-formula.sh
+
+      - name: Commit formula update
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: chore(homebrew): sync formula for release
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)